Kernel_23/test/Kernel_23/test_approximate_dihedral_angle_3.cpp

```diff
@@ -53,7 +53,6 @@ int main() {
 
   Point_3 a = {0, 0, 0};
   Point_3 b = {0, -1, 0}; // ab is oriented so that it sees the plan xz positively.
-  [[maybe_unused]] Point_3 c = {1, 0, 0};
   // c can be any point in the half-plane xy, with x>0
```

Comment:
Shouldn't comment be removed as well (as `c` is a bit hanging in thin air now)?

Reply:
c is defined below which is why I left the comment.

Reply:
Sure it is below and I saw it as well, but for the quick reader this is not clear.
Maybe a reference to the variable `c` in the loop is in place here.
It is good that `c` has been removed as it was hiding the real one.